tests/foreman/cli/test_template.py

```diff
@@ -3,7 +3,11 @@
 """Test class for Template CLI"""
 
 from fauxfactory import gen_string
-from robottelo.cli.factory import CLIFactoryError, make_template, make_os
+from robottelo.cli.factory import (
+    CLIFactoryError,
+    make_location, make_org,
+    make_os, make_template
```

Comment:
Place each import on its own line. Doing so makes it easier to visually scan down the list to see what's imported, makes diffs easier to review, and makes it possible to sort imports. (In vim: `7G:.,.4 sort`.)